packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -50,67 +100,78 @@ export async function getConsolidatedPatientData({
 
   const fhir = makeFhirApi(cxId);
   const errorsToReport: Record<string, string> = {};
-  const searchResources = async <K extends ResourceType>(
-    resource: K,
-    searchFunction: () => AsyncGenerator<ExtractResource<K>[]>
-  ) => {
-    try {
-      const pages: Resource[] = [];
-      for await (const page of searchFunction()) {
-        pages.push(...page);
-      }
-      return pages;
-      //eslint-disable-next-line @typescript-eslint/no-explicit-any
-    } catch (err: any) {
-      if (err instanceof OperationOutcomeError && err.outcome.id === "not-found") throw err;
-      if (err instanceof OperationOutcomeError) errorsToReport[resource] = getMessage(err);
-      else errorsToReport[resource] = err.message;
-      throw err;
-    }
-  };
```

Comment:
Moved to [its own function](https://github.com/metriport/metriport/pull/820/files#diff-01fe75170935f9bd9b4e63bd38b2ed0e073788084ede3e2677ccf8b0671e1fbcR156) to simplify this one